### DISCUSSION TITLE
Queue Processing

### DISCUSSION THREAD

**Discussion by vijay-2001** [rabbitmq/rabbitmq-server#10063] [Questions]
I am facing a issues in queue as it is in idle state. Certain queues are waiting for long time to process and some are completed

**Comment by michaelklishin** [comment]
@vijay-2001 a queue will report its state as idle when there are no ingress messages or deliveries to consumers. In other words, when it does not do anything besides collecting metrics, during a certain time window.

We cannot help with a single sentence problem definition. You need to collect and provide a lot more relevant metrics, and possibly a traffic capture, or no one would be able to understand what is going on in your system. Please help others help you.

Some of your publishers may be publishing messages in a way that [makes them routed nowhere](https://rabbitmq.com/publishers.html#unroutable) or consumers may be [consuming with a prefetch of one](https://rabbitmq.com/confirms.html#channel-qos-prefetch) and never acknowledging messages [until a delivery timeout hits](https://rabbitmq.com/consumers.html#acknowledgement-timeout).

Some of your consumers may lose connection and [never reconnect](https://rabbitmq.com/api-guide.html#recovery).

We do not guess in this community.
